src/widgetastic_patternfly/__init__.py

```diff
@@ -1888,6 +1888,69 @@ def items(self):
         return items
 
 
+class Modal(View):
+    """ Patternfly modal widget
+
+        https://www.patternfly.org/pattern-library/widgets/#modal
+    """
+    ROOT = ('.//div[contains(@class, "modal") '
+            'and contains(@class, "fade") and @role="dialog"]')
+
+    def __init__(self, parent, id=None, logger=None):
+        if id:
+            self.id = id
+            self.ROOT = ParametrizedLocator(
+                './/div[normalize-space(@id)={@id|quote} and '
+                'contains(@class, "modal") and contains(@class, "fade") '
+                'and @role="dialog"]')
+
+        View.__init__(self, parent, logger=logger)
+
+    @property
+    def title(self):
+        return self.header.title.read()
+
+    @property
+    def is_displayed(self):
+        """ Is the modal currently open? """
+        try:
+            return "in" in self.browser.classes(self)
+        except NoSuchElementException:
+            return False
+
+    def close(self):
+        """Close the modal"""
+        self.header.close.click()
```

Comment:
I think it would be good to wrap this in is_displayed, so that its 'safe' to call when the modal has maybe unexpectedly closed.

@izapolsk thoughts? I can add a commit for this if you agree, since john is on vacation.

Reply:
I agree. I'm currently fixing one compatibility issue. so, I'll fix this one as well.